app/src/main/java/io/left/meshim/database/MeshIMDao.java

```diff
@@ -77,12 +80,13 @@
      *     direction because we will not have exchanged a message with every user in the
      *     Users table.
      * </p>
-     * @return a summary of every conversation the device's user has started
+     * @return a summary of every conversation the device's user has started, and number of unread messages
```

Comment:
This description is probably fine as is - it doesn't return a list and a value, just the list. The number of unread messages is included in the description.